### PR TITLE
[Art] Yet another copper wire

### PR DESCRIPTION
**Pull request by dusho**
pixel arty copper wire that should be consistent with other copper things
image
if nothing.. it at least removes old 64kB image of copper wire for new one that is 1kB

### CODE REVIEW

**Comment by Tranberry** [comment]
A real improvement imo I approve!

File: Assets/StreamingAssets/Data/Inventory.xml [hunk 1]
```diff
@@ -7,4 +7,5 @@
     <Inventory type="Steel Plate" maxStackSize="50" basePrice="1.5" category="inv_cat_refined"/>
     <Inventory type="Power Cell" maxStackSize="5" basePrice="5" category="inv_cat_crafted"/>
     <Inventory type="Copper Plate" maxStackSize="50" basePrice="1.5" category="inv_cat_refined"/>
+	<Inventory type="Copper Wire" maxStackSize="50" basePrice="2" category="inv_cat_refined"/>
```
Comment (TomMalbran) [inline Assets/StreamingAssets/Data/Inventory.xml]: Use spaces please.

Comment (dusho) [inline Assets/StreamingAssets/Data/Inventory.xml]: fixed

**Comment by ComputerOverlord** [comment]
It's in perspective, too!

**Comment by koosemose** [comment]
There've been no significant complaints against this one, and as the one it is replacing is drastically outside of the artstyle, I'll go ahead and merge this. Merged.

**Pull request merged**